hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -510,6 +499,14 @@
       base_addrs: {core: "0x41000000", prim: "0x41008000"}
       attr: "templated",
     },
+    { name: "rv_dm",
+      type: "rv_dm",
+      clock_srcs: {clk_i: "main"},
+      clock_group: "secure",
+      reset_connections: {rst_ni: "lc"},
```

Comment:
does this reset and clock group look right to you, @tjaychen?

Reply:
yes this looks good. The only thing is i think the clock group is probably technically `infra`, but it doesn't really make a difference in this case.

Reply:
I can make it "infra" for consistency...